Makefile

```diff
@@ -19,12 +19,14 @@ build: build-ig build-gadget-container
 .PHONY: build-ig
 build-ig: kubectl-gadget-linux-amd64 kubectl-gadget-darwin-amd64
 
+.PHONY: kubectl-gadget-linux-amd64
```

Comment:
But it is not a phony target, the file `kubectl-gadget-linux-amd64` is really created by this target.

https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
> A phony target is one that is not really the name of a file

I guess you did this so that it would always recompile it? In this case, the target name could be also renamed to `build-ig-linux-amd64`. Wdyt?

Reply:
It's better to rename and keep it as a PHONY target. I added that because I did a change and run `make kubectl-gadget-linux-amd64` and nothing happens because the file was there.